iris-mpc-common/src/job.rs

```diff
@@ -64,6 +64,8 @@ pub struct ServerJobResult {
     pub successful_reauths: Vec<bool>, // true if request type is reauth and it's successful
     pub reauth_target_indices: HashMap<String, u32>,
     pub reauth_or_rule_used: HashMap<String, bool>,
+    /// Actor-specific data (e.g. graph mutations).
+    pub actor_data: A,
```

Comment:
I think this looks fine, just a design question:
Should we aim for the db insertion stuff to also be internal to the actor? This again would now add code that is unique to the CPU impl to the `server_hawk.rs`

Reply:
Good point to try to minimize change in `server*.rs`. As it stands the GraphStore needs a plain old data structure from the Hawk search, but a complex interaction with the startup, database update, and crash recovery mechanisms.

Reply:
I got it down to pure method calls - one for startup loading and one for updates.